content/en/security_platform/cloud_siem/signal_correlation_rules.md

```diff
@@ -0,0 +1,156 @@
+2. Detect if there was an attempt to authenticate into a host or resource
+
+And use the `expired account ID` attribute to correlate the two rules. 
+
+You can correlate log detection and signal correlation rules, as well as log detection rules with Cloud Workload Security and Application Security Monitoring rules.
+
+## Create a new Signal Correlation rule
+
+Go to [Detection Rules][1] and click **+ New Rule**. In the *Select a rule type* section, click **Signal Correlation**.
+
+### Set rules
+
+Select a rule for *Rule a*. Click the pencil icon to rename the rule. Use the *correlated by* dropdown to define the correlating attribute. You can select multiple attributes (maximum of 3) to correlate the selected rules. See [Time windows](#time-windows) for more information about the sliding window.
+
+Select a rule for *Rule b* in the second Rule editor’s dropdown. Click the pencil icon to rename the rule. The attributes and sliding window time frame is set to what was selected for *Rule a*.
+
+### Set rule cases
+
+#### Trigger
+
+{{< img src="security_platform/security_monitoring/detection_rules/define_rule_case.png" alt="The set rule cases section showing the trigger, severity, and notification fields" >}}
+
+Rule cases are evaluated as case statements. Thus, the first case to match generates the signal. An example of a rule case is`a > 3`, where `a` is the rule name. Click and drag your rule cases to manipulate their ordering.
+
+A rule case contains logical operations (`>, >=, &&, ||`) to determine if a signal should be generated based on the event counts in the previously defined queries. The ASCII lowercase [rule names](#set-rules) are referenced in this section.
+
+**Note**: The query label must precede the operator. For example, `a > 3` is allowed; `3 < a` is not allowed.
+
+Provide a **name**, for example "Case 1", for each rule case. This name is appended to the rule name when a signal is generated.
+
+#### Severity and notification
+
+Set the severity of the Security Signal. The dropdown allows you to select an appropriate severity level (`INFO`, `LOW`, `MEDIUM`, `HIGH`, `CRITICAL`).
+
+In the “Notify” section, configure zero or more [notification targets][2] for each rule case.
+
+#### Time windows
+
+An `evaluation window` is specified to match when at least one of the cases matches true. This is a sliding window and evaluates in real-time.
+
+Once a signal is generated, the signal will remain “open” if a case is matched at least once within this `keep alive` window. Each time a new event matches any of the cases, the *last updated* timestamp is updated for the signal.
+
+A signal will “close” regardless of the query being matched once the time exceeds the `maximum signal duration`. This time is calculated from the first seen timestamp.
+
+Click **Add Case** to add additional cases.
+
+**Note**: The `evaluation window` must be less than or equal to the `keep alive` and `maximum signal duration`.
```

Comment:
nit: I know it is a sentence retrieve from the existing documentation about `Log detection`, but should we specify `keep alive` must also be less than or equal to the `maximum signal duration`.

Reply:
i can remove the sentence from both docs if you want. Can `keep alive` be greater than `maximum signal`?